core/src/main/java/com/linecorp/armeria/internal/server/CorsHeaderUtil.java

```diff
@@ -0,0 +1,146 @@
+package com.linecorp.armeria.internal.server;
```

Comment:
Could you add the copyright header?

Reply:
@trustin 
I added copyright headers, but the linter didn't recognize it.
If you know, could you tell me the way to solve it?

Reply:
It seems that linter doesn't work correctly with using latest copyright header（`Copyright 2024 LY Corporation`）.
However, I used previous copyright header `Copyright 2024 LINE Corporation` Instead of it, the linter test passed.
 https://armeria.dev/community/developer-guide/

Reply:
Let's make the linter happy for now. We'll fix them later 😅